### DISCUSSION TITLE
Add a hotkey/button to reset slider velocity (to 1x) for the next placed slider

### DISCUSSION THREAD

**Discussion by schiWasTaken** [ppy/osu#29289] [Ideas]
In the osu!stable editor, you can adjust slider velocities using the timing menu at any time. However in lazer, you can only adjust slider velocities after making a slider, and it can be quite clunky if the previous slider's velocity is really high as you can't make a slider that's shorter than the beat division ticks. It's not impossible as I have found a workaround (shown in the video attached) but I still find it quite unintuitive. It would be nice if I can just quickly press a button to reset the SV to 1x and not worry about sliders being too short. An alternative solution is to have the hotkey open a menu for you to type in the desired slider velocity.

https://github.com/user-attachments/assets/8db96d80-6b85-46b8-884b-c74623e85fb2

**Comment by felipemarins** [comment]
I think the proposed alternative solution is much more appropriate and could be used in more scenarios. As an example, I think it could also work as a faster way of changing slider velocity when you have a slider selected. When you don't have any slider selected, it would change the slider velocity used on the creation of a new slider, as proposed above.

**Reply by joppyloppy** [reply]
i also agree with the proposed alternative solution of having a hotkey or keybind to edit the slider velocity of selected objects

**Comment by kstefanowicz** [comment]
That workaround is kinda hilarious but actually seems to be a functional solution. Adding a hotkey to open that menu or reset to 1x could help, but I can't imagine another solution that wouldn't be similarly awkward

**Comment by peppy** [comment]
Slider velocity should have a slider in the right inspector area.

**Reply by felipemarins** [reply]
So, would it be possible for a slider being created to appear on the inspector area?

**Reply by schiWasTaken** [reply]
i was thinking maybe add a dropdown button on the slider tool button itself, cause the right inspector menu feels like it's more for note placement related settings, but i guess it wouldn't hurt for slider velocity to be there
